tests/tests-main.cc

```diff
@@ -0,0 +1,73 @@
+/** HELPER FILE FOR TESTS.CC ***/
+
+#include "../src/rule.h"
+#include "../src/utils.h"
+
+#define CATCH_CONFIG_RUNNER
+#include "catch.hpp"
+
+rule_t * rules;
+rule_t * labels;
+rule_t * minority;
+int nrules;
+int nsamples;
+int nlabels;
+int nminority;
+
+NullLogger * logger;
+
+int main(int argc, char* argv[])
+{
+    rules = NULL;
+    labels = NULL;
+
+    int verbosity = 0;
+
+    if(argc > 1 && strcmp(argv[argc-1], "-v") == 0) {
```

Comment:
Does the order matter for the verbosity flag?

Reply:
Yes, because you might want to provide arguments to Catch, which must come before the -v argument (which has nothing to do with Catch).